sonic-xcvrd/xcvrd/xcvrd_utilities/port_event_helper.py

```diff
@@ -119,9 +129,19 @@ def handle_port_update_event(self):
                     (port_name, op, fvp) = port_tbl.pop()
                     if not port_name:
                         break
-                    if not validate_port(port_name):
-                        continue
+ 
                     fvp = dict(fvp) if fvp is not None else {}
+                    role = fvp.get(multi_asic.PORT_ROLE, None)
```

Comment:
@vivekrnv Why not always get the `role` from self.port_role_map{} which is updated in the __init__ with the role form config DB?

Reply:
This is to handle DPB use case